server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSOptionsCheckBaseUtility.java

```diff
@@ -170,6 +165,42 @@ private void throwException(
     }
   }
 
+  private void checkDuplicateEntries(
+      ParserRuleContext ctx,
+      Map<Integer, TerminalNode> entries,
+      Map<String, ErrorSeverity> specialSeverities) {
+    if (ctx.getChildCount() != 0) {
+      for (ParseTree entry : ctx.children) {
+        if (entry.getChildCount() == 0) {
+          TerminalNode current = (TerminalNode) entry;
+          if (current.getSymbol().getType() < 897) {
+            if (entries.containsKey(current.getSymbol().getType())) {
+              ErrorSeverity severity = ErrorSeverity.ERROR;
+              if (specialSeverities.containsKey(current.getText())) {
```

Comment:
I'd expect the getText to return a string that might not be upper-cased, breaking the severity lookup.

Also you can `getOrDefault`.